packages/wonder-blocks-search-field/src/components/search-field.stories.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import * as React from "react";
+
+import type {StoryComponentType} from "@storybook/react";
+
+import SearchField from "../index.js";
+
+export default {
+    title: "SearchField",
+};
+
+export const Default: StoryComponentType = () => {
```

Comment:
There are a bunch of other variants that should probably be showcased as well, e.g. `disabled`, `light`, something with custom styling?  Once the flow issue is fixed and the chromatic action starts working again it'll snapshot each of the stories.  For some of the other components we minimize the number of snapshots by having one story that shows a whole bunch of variants together and then we disable chromatic on all of the stories except for the one with multiple variants.